recipes/libudev/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+import os
+
+
+class LibudevConan(ConanFile):
+    name = "libudev"
+    version = "system"
+    description = "API for enumerating and introspecting local devices"
+    topics = ("conan", "udev")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://systemd.io/"
+    license = "LGPL-2.1-or-later"
+    settings = {"os": "Linux"}
+
+    def system_requirements(self):
+        if tools.os_info.is_linux and self.settings.os == "Linux":
+            package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
+            if tools.os_info.with_yum or tools.os_info.with_dnf:
+                packages = ["libudev-devel"]
+            elif tools.os_info.with_apt:
+                packages = ["libudev-dev"]
+            elif tools.os_info.with_pacman:
+                packages = ["libsystemd", "systemd"]
```

Comment:
```suggestion
                packages = ["libsystemd"]
```

Reply:
The systemd package is necessary because it contains the headers. libsystemd only contains .so files.

Reply:
Right! https://www.archlinux.org/packages/core/x86_64/systemd-libs/